Fw/Interfaces/ChannelInterfaces.fppi

```diff
@@ -0,0 +1,2 @@
+@ Telemetry port
+telemetry port tlmOut 
```

Comment:
Last step is to delete line 68-69 in `SignalGen.fpp` as your interface adds that port!